src/server.py

```diff
@@ -554,6 +560,21 @@ def play_sound(self, sound_event_id, connection, sender, path, iface,
 
         return invocation.return_value(GLib.Variant('(s)', (uuid_, )))
 
+    def _check_too_many_sounds(self, invocation, sound_event_id,
+                               overlap_behavior):
+        n_instances = len(self._uuid_by_event_id[sound_event_id])
+        if n_instances < self._MAX_SIMULTANEOUS_SOUNDS:
+            return False
+        self.logger.warning("Attempting to play this sound more than "
+                            "%d times. Forbidden.",
+                            self._MAX_SIMULTANEOUS_SOUNDS,
+                            sound_event_id=sound_event_id)
+        invocation.return_dbus_error(
```

Comment:
This should not be an error. (There's no way for the caller to predict and avoid it.) The sound should just be silently dropped, as I think situations where more than 5 sounds are played are just going to be part of normal operation.

Reply:
what should I return?

Reply:
oh, I see, you have to return a UUID. that's difficult.

I can think of two possibilities, either return an invalid UUID (such as a blank string), and ignore any requests to stop that UUID; or return a valid UUID and store it in a dict of non-playing UUIDs, and ignore any requests to stop it.

I don't like either of these suggestions very much, but I think the second one is worse, because there is no condition on which we can remove UUIDs from the dict.